REGRESSION.md

```diff
@@ -169,21 +165,6 @@ Babbage uses the next publish date to calculate a `Cache-Control` `max-age` head
 
 Babbage calculates a hash of the content response to serve as an eTag header.
 
-### Indexing search
```

Comment:
Also remove the `Cache times` section above as it references the publish_dates index